packages/universal-components/src/RangeDatePicker/RangeDateConfig.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+export default {
+  dayItemHeight: 52,
+  dayItemPadding: 8,
+  numberOfRenderedMonths: 24,
```

Comment:
Maybe we can decrease this number just to one year. But it is totally up to you. Even two years are ok if it no longer affects the performance.

Reply:
When it is lazy-calculated, it should not matter 😊 

I see that on web you can select as far into the future as you want, while on mobile there is just one year into the future.

Reply:
Ok, it probably depends more on our API and how far into future can really return any results.
So we can keep 24 for now or maybe we can allow this value to be defined by props to make component even more flexible.

Reply:
> defined by props to make component even more flexible

Absolutely the best option I would say

Reply:
It's true. Done